packages/web-integration/src/configuration.ts

```diff
@@ -0,0 +1,2 @@
+export const apiBaseUrl = "https://api.hypernet.id";
```

Comment:
We probably want these to be passed as arguments so we could have different variables for different environments

Reply:
This package is explicitly for public usage; the only environment that it should be used against is production, but point taken. I will look at making these values overrideable.